pom.xml

```diff
@@ -596,6 +656,23 @@
                     <dependencyLocationsEnabled>false</dependencyLocationsEnabled>
                 </configuration>
             </plugin>
+            <plugin>
+                <artifactId>maven-checkstyle-plugin</artifactId>
+                <configuration>
+                    <skip>true</skip>
```

Comment:
:question: What's the point of having a plugin run configured that always skips?

Reply:
This is the reporting section, so I believe it prevents checkstyle from creating a report file.
That is conjecture; this block of code got moved around so it is marked as "changed", but skip was true previously.